tests/test_bot.py

```diff
@@ -2881,6 +2890,11 @@ async def test_set_webhook_get_webhook_info_and_delete_webhook(self, bot, use_ip
         assert info.ip_address is None
         assert info.has_custom_certificate is False
 
+    async def test_get_business_connection(self, bot):
+        # TODO: Get a business connection and test this properly
+        pass
+        # assert await bot.get_business_connection("TEST") is None
+
```

Comment:
I don't think we'll be able to do that without having to subscribe to TG business. We can monkeypatch HTTPXRequset.do_request to return a json payload, that's probably as close as we can reasonably get 🤔